tests/plugins/test_add_labels.py

```diff
@@ -296,16 +296,30 @@ def test_add_labels_aliases(tmpdir, docker_tasker, caplog,
     if expected_log:
         assert expected_log in caplog.text()
 
-@pytest.mark.parametrize('labels', [{"distribution-scope": "restricted",
-                                     "dont_overwrite_if_in_dockerfile": "distribution-scope",
-                                     "auto_labels": [], 'aliases': {}},
-                                    {"distribution-scope": "restricted",
-                                     "auto_labels": [], 'aliases': {}}
+@pytest.mark.parametrize('parent_scope, docker_scope, result_scope, overwrite', [
+    (None, None, "restricted", False),
+    ("public", None, "restricted", False),
+    ("private", None, "restricted", False),
+    ("restricted", 'public', "public", False),
+    ("restricted", 'restricted', "restricted", False),
+    ("restricted", 'private', "private", False),
+    (None, None, "restricted", True),
+    ("public", None, "restricted", True),
+    ("private", None, "restricted", True),
+    ("restricted", 'public', "public", True),
+    ("restricted", 'restricted', "restricted", True),
+    ("restricted", 'private', "private", True)
 ])
-def test_dont_overwrite_distribution_scope(tmpdir, docker_tasker, labels):
+def test_dont_overwrite_distribution_scope(tmpdir, docker_tasker, parent_scope,
+                                           docker_scope, result_scope, overwrite):
```

Comment:
This all looks great, but the sense of `overwrite` seems to be inverted. I'd expect it to either be named `dont_overwrite` or have opposite True/False values.